### PR TITLE
Add github Actions

### PR DESCRIPTION
**Pull request by Dr1nkingbird**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        BUILD_TYPE: ["Debug"]
+        os: [ windows-latest ]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get MSVC
```
Comment (youyuanwu) [inline .github/workflows/build.yml]: This is not required since we write rust not c++

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - name: Get specific version CMake, v3.21.2
+      uses: lukka/get-cmake@v3.21.2
+
+    - name: Install rust stable
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    # need to use the latest winrt in sdk, i.e. pass the version to cmake
+    - name: find latest win sdk
```
Comment (youyuanwu) [inline .github/workflows/build.yml]: This is not needed.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      run: |
+          $sdkver = Get-ChildItem "Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots\" | Select-Object -Last 1 -ExpandProperty PSChildName
+          echo "Using win sdkver $sdkver"
+          echo $env:GITHUB_ENV
+          echo "sdkver=$sdkver" >> $env:GITHUB_ENV
+      shell: pwsh
+
+    - name: run cmake
+      env:
+        BOOST_ROOT: ${{ steps.install-boost.outputs.BOOST_ROOT }}
```
Comment (youyuanwu) [inline .github/workflows/build.yml]: This env var is not needed.

**Review by youyuanwu — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (Dr1nkingbird) [reply .github/workflows/build.yml:15]: I think this is for the requirement: "service fabric runtime installation"

File: .github/workflows/build.yml (hunk 2 above)
Comment (Dr1nkingbird) [reply .github/workflows/build.yml]: I think this is for the requirement "service fabric runtime installation"

Comment (Dr1nkingbird) [reply .github/workflows/build.yml]: link: https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started

Comment (youyuanwu) [reply .github/workflows/build.yml]: Sdk is installed in action machines by default. There is no need to query the version.
The original code was need to configure cpp headers.

**Review by youyuanwu — CHANGES_REQUESTED** [review]

**Comment by Dr1nkingbird** [comment]
[https://github.com/youyuanwu/fabric-rpc-rs/issues/1](url)

File: .github/workflows/build.yml (hunk 2 above)
Comment (Dr1nkingbird) [reply .github/workflows/build.yml]: Removed and it works well. Thanks youyuan

**Review by youyuanwu — APPROVED** [review]

**Pull request merged**